### PR TITLE
Fix/docker

### PR DESCRIPTION
**Pull request by haraprasadj**
Updated Dockerfile
Added docker-compose for local development
Fixed some dependencies

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,20 +1,23 @@
...
-COPY --from=builder /root/.poetry /root/.poetry
 COPY --from=builder /env /env
 COPY --from=builder /src /src
 ENV PATH="/env/bin/:${PATH}"
 WORKDIR /src
-CMD ["sh", "startup_script.sh"]
+
+CMD ["/env/bin/gunicorn", "thor.main:app", "-b", "0.0.0.0:80", "-k", "uvicorn.workers.UvicornWorker"]
+CMD psql -U postgres -c "create database thor_test_tmp"
+CMD python thor/create_all_tables.py
```
Comment (colinyaos) [inline Dockerfile:23]: I'm getting an error where Python can't open this thor/create_all_tables. Changing this to src/thor/create_all_tables solves this problem, but I'm not sure what other side effects it has.

Comment (haraprasadj) [reply Dockerfile]: those lines are accidentally pasted here, removed the redundant lines, please try now

**Review by colinyaos — COMMENTED** [review]
Not sure if these are redundant, as otherwise, we do not create the database and the tables within it for storing data. Currently getting a "database "thor_test_tmp" does not exist" error that I think is due to this

**Comment by haraprasadj** [comment]
> Not sure if these are redundant, as otherwise, we do not create the database and the tables within it for storing data. Currently getting a "database "thor_test_tmp" does not exist" error that I think is due to this

We should move this part to the tests, we should only create this db while running tests, not in production. I added some docs to run locally using Docker here - https://github.com/uc-cdis/thor/blob/59d840f868f90e93fff44e737ebf8b6c61e5a767/README.md#local-development-with-docker

**Comment by colinyaos** [comment]
Ok, everything looks good from here. +1

**Review by colinyaos — APPROVED** [review]

**Pull request merged**